.travis.yml

```diff
@@ -37,6 +37,13 @@ before_script:
     # set up CORS
   - "./node_modules/.bin/add-cors-to-couchdb http://127.0.0.1:3000"
 
+    # Configure CORS against master
+  - curl -X PUT http://127.0.0.1:3001/_node/node1@127.0.0.1/_config/httpd/enable_cors -d '"true"'
+  - curl -X PUT http://127.0.0.1:3001/_node/node1@127.0.0.1/_config/cors/origins -d '"*"'
+  - curl -X PUT http://127.0.0.1:3001/_node/node1@127.0.0.1/_config/cors/credentials -d '"true"'
+  - curl -X PUT http://127.0.0.1:3001/_node/node1@127.0.0.1/_config/cors/methods -d '"GET, PUT, POST, HEAD, DELETE"'
+  - curl -X PUT http://127.0.0.1:3001/_node/node1@127.0.0.1/_config/cors/headers -d '"accept, authorization, content-type, origin, referer, x-csrf-token"'
+
```

Comment:
Can you please open a PR on `add-cors-to-couchdb` add this as well? It's been a much-requested feature in that tool for a long time.